satpy/etc/readers/thr3mi_l1c_nc.yaml

```
@@ -0,0 +1,299 @@
+reader:
+  name: thr3mi_l1c_nc
+  short_name: 3MI L1C RAD NetCDF4
+  long_name: EPS-SG 3MI L1C Radiance (NetCDF4)
+  description: >
+    Reader for EUMETSAT EPS-SG 3MI Multi-Angle Polarimeter Level 1C Radiance files in NetCDF4 format per PFS
+  sensors: [3mi]
+  reader: !!python/name:satpy.readers.yaml_reader.FileYAMLReader
+
```

a couple more fields are required nowadays
```suggestion
  supports_fsspec: true
  status: Beta
```